sycl/include/sycl/ext/oneapi/experimental/invoke_simd.hpp

```diff
@@ -37,10 +37,11 @@
 /// @tparam SimdCallee the type of the SIMD callee function (the "target"). Must
```

Comment:
I think this comment needs rewording. Thanks

Reply:
@asudarsa - thank you for the review.
Good catch. I updated the comment in this commit: 8c0c71b21a62f5399a3ed68090ea4523d02134be